sources/packages/web/src/components/generic/DesignationStatusBadge.vue

```diff
@@ -0,0 +1,36 @@
+<!--
+Used when we need to display the designation status for an Institution
+-->
+<!-- TODO: Replace v-badge with vuetify2 equivalent v-chip with icon once veutify3 is released-->
```

Comment:
@guru-aot this is vuetify3 component, no need to replace it, you can remove the comment